armi/bookkeeping/report/reportingUtils.py

```diff
@@ -264,7 +264,8 @@ def writeAssemblyMassSummary(r):
         thisTypeList = core.getChildrenOfType(a.getType())
         count = 0
         for t in thisTypeList:
-            if t.getLocationObject().i1 == 1:
+            ring = t.spatialLocator.getRingPos()[0]
```

Comment:
I find `ring, _pos = t.spatialLocator.getRingPos()` to be slightly more self-documenting. nbd though